### PR TITLE
Set gpkg as BINPKG_FORMAT default

### PR DESCRIPTION
**Pull request by akhuettel**
**Untested so far**, but pretty straightforward given existing mechanisms

### CODE REVIEW

**Comment by thesamesam** [comment]
Did you have a chance to test this yet?

**Comment by akhuettel** [comment]
Needed an entry in meson.build, but otherwise works now as intended. Here's the required patch for portage-9999.ebuild : 

```
diff --git a/sys-apps/portage/portage-9999.ebuild b/sys-apps/portage/portage-9999.ebuild
index 5bf30b190b87e..abf843edb7357 100644
--- a/sys-apps/portage/portage-9999.ebuild
+++ b/sys-apps/portage/portage-9999.ebuild
@@ -205,6 +205,10 @@ pkg_preinst() {
                env -u FEATURES -u PORTAGE_REPOSITORIES \
                        PYTHONPATH="${D}${sitedir}${PYTHONPATH:+:${PYTHONPATH}}" \
                        "${PYTHON}" -m portage._compat_upgrade.binpkg_multi_instance || die
+
+               env -u BINPKG_FORMAT \
+                       PYTHONPATH="${D}${sitedir}${PYTHONPATH:+:${PYTHONPATH}}" \
+                       "${PYTHON}" -m portage._compat_upgrade.binpkg_format || die
        fi
 
        # elog dir must exist to avoid logrotate error for bug #415911.
```

**Comment by akhuettel** [comment]
Just as a remark, this has, ahem, interesting side effects if binpkg-request-signature is enabled but local binpackage signing is not set up. You suddenly can't generate any binary packages locally anymore!

**Comment by thesamesam** [comment]
> Just as a remark, this has, ahem, interesting side effects if binpkg-request-signature is enabled but local binpackage signing is not set up. You suddenly can't generate any binary packages locally anymore! (Trying to do so results in a backtrace and invalid files in the package directory.)

I think this was always a problem if you set `binpkg-request-signature` -- i.e. it's not new to the default change, it just makes it more prominent? Or am I misunderstanding?

File: lib/portage/_compat_upgrade/meson.build [hunk 1]
```diff
@@ -1,6 +1,7 @@
 py.install_sources(
     [
         'binpkg_compression.py',
+	'binpkg_format.py',
```
Comment (zmedico) [inline lib/portage/_compat_upgrade/meson.build:4]: Indentation is abnormal here.

**Review by zmedico — APPROVED** [review]

File: lib/portage/_compat_upgrade/meson.build (hunk 1 above)
Comment (thesamesam) [reply lib/portage/_compat_upgrade/meson.build:4]: Fixed on merge

**Pull request closed without merging**